apis/core/lib/domain/dataset/update.ts

```diff
@@ -41,6 +43,7 @@ export async function update({
     .addOut(rdf.type, dcat.Dataset)
 
   hasPart.forEach(child => datasetResource.addOut(schema.hasPart, child))
+  dimensionMetadata.forEach(child => datasetResource.addOut(cc.dimensionMetadata, child))
```

Comment:
You don't need to modify this file. The dataset resource is in a different graph from the metadata

Reply:
But the triple `dataset cc.dimensionMetadata metadatacollection` has been deleted by the deleteOut by an update in the end to end tests.

Reply:
Aha, my mistake. I mistook these for the `schema:hasPart` triples. Sorry